sdks/java/io/kafka/src/main/java/org/apache/beam/sdk/io/kafka/KafkaIO.java

```diff
@@ -1062,8 +1069,10 @@ public Instant getWatermark() {
         return initialWatermark;
       }
 
+      //watermark increases as current_timestamp, in case of latency,
+      //developers need to either provide a customized watermarkFn, or tune Triggers.
       return source.spec.getWatermarkFn() != null
-          ? source.spec.getWatermarkFn().apply(curRecord) : curTimestamp;
```

Comment:
I set the default watermark to current_timestamp if no user-specified `WarermarkFn`. This should be the same as before since `curTimestamp` in kafka0.9 equals to current_timestamp if no `timestampFn` provided.

I guess in kafka0.9, if `timestampFn` is provided, this `watermark` cannot guarantee to monotone increasing.